config/jobs/image-pushing/k8s-staging-sig-storage.yaml

```diff
@@ -19,7 +19,7 @@ postsubmits:
       spec:
         serviceAccountName: gcb-builder
         containers:
-          - image: gcr.io/k8s-testimages/image-builder:v20210204-b06ec78
+          - image: gcr.io/k8s-testimages/image-builder:v20200901-ab141a0
```

Comment:
I think you'll need to update k8s-staging-sig-storage.sh with the new image-builder tag. Unfortunately the script that auto updates the image version doesn't update the script that generates this yaml.

Reply:
https://github.com/kubernetes/test-infra/pull/20960 should make autobump change the script going forward